modules/mod_base/controllers/controller_postback.erl

```diff
@@ -28,6 +28,7 @@
 
 -include_lib("zotonic.hrl").
 
+-define(SIDEJOB_TIMEOUT, 1000).
```

Comment:
Aha... I see... but the ack just means that the message arrived, not that it was processed correctly.